mcumgr-core/src/test/java/io/runtime/mcumgr/response/McuMgrResponseTest.java

```diff
@@ -0,0 +1,325 @@
+                (byte) 0x30, (byte) 0x15, (byte) 0xA2, (byte) 0xDA, (byte) 0xE6, (byte) 0x38, (byte) 0x73, (byte) 0x99, (byte) 0xF0,
+                (byte) 0x20, (byte) 0x1C, (byte) 0x3C, (byte) 0x0C, (byte) 0x56, (byte) 0x15, (byte) 0xF0, (byte) 0xDD, (byte) 0x7A,
+                (byte) 0x2D, (byte) 0x49, (byte) 0xCD, (byte) 0xF3, (byte) 0x46, (byte) 0xB2, (byte) 0x68, (byte) 0x62, (byte) 0x6F,
+                (byte) 0x6F, (byte) 0x74, (byte) 0x61, (byte) 0x62, (byte) 0x6C, (byte) 0x65, (byte) 0xF5, (byte) 0x67, (byte) 0x70,
+                (byte) 0x65, (byte) 0x6E, (byte) 0x64, (byte) 0x69, (byte) 0x6E, (byte) 0x67, (byte) 0xF4, (byte) 0x69, (byte) 0x63,
+                (byte) 0x6F, (byte) 0x6E, (byte) 0x66, (byte) 0x69, (byte) 0x72, (byte) 0x6D, (byte) 0x65, (byte) 0x64, (byte) 0xF5,
+                (byte) 0x66, (byte) 0x61, (byte) 0x63, (byte) 0x74, (byte) 0x69, (byte) 0x76, (byte) 0x65, (byte) 0xF5, (byte) 0x69,
+                (byte) 0x70, (byte) 0x65, (byte) 0x72, (byte) 0x6D, (byte) 0x61, (byte) 0x6E, (byte) 0x65, (byte) 0x6E, (byte) 0x74,
+                (byte) 0xF4, (byte) 0xFF, (byte) 0xBF, (byte) 0x64, (byte) 0x73, (byte) 0x6C, (byte) 0x6F, (byte) 0x74, (byte) 0x01,
+                (byte) 0x67, (byte) 0x76, (byte) 0x65, (byte) 0x72, (byte) 0x73, (byte) 0x69, (byte) 0x6F, (byte) 0x6E, (byte) 0x65,
+                (byte) 0x33, (byte) 0x2E, (byte) 0x30, (byte) 0x2E, (byte) 0x30, (byte) 0x64, (byte) 0x68, (byte) 0x61, (byte) 0x73,
+                (byte) 0x68, (byte) 0x58, (byte) 0x20, (byte) 0x3A, (byte) 0xA1, (byte) 0x47, (byte) 0xC6, (byte) 0x3C, (byte) 0xD8,
+                (byte) 0xE8, (byte) 0x71, (byte) 0x10, (byte) 0x4F, (byte) 0x7E, (byte) 0xBB, (byte) 0x1C, (byte) 0xD0, (byte) 0xB2,
+                (byte) 0xE7, (byte) 0x42, (byte) 0xD8, (byte) 0x3D, (byte) 0xD7, (byte) 0x9A, (byte) 0x58, (byte) 0x1C, (byte) 0xB5,
+                (byte) 0xAE, (byte) 0x0E, (byte) 0x05, (byte) 0xDB, (byte) 0x79, (byte) 0x44, (byte) 0x47, (byte) 0x99, (byte) 0x68,
+                (byte) 0x62, (byte) 0x6F, (byte) 0x6F, (byte) 0x74, (byte) 0x61, (byte) 0x62, (byte) 0x6C, (byte) 0x65, (byte) 0xF5,
+                (byte) 0x67, (byte) 0x70, (byte) 0x65, (byte) 0x6E, (byte) 0x64, (byte) 0x69, (byte) 0x6E, (byte) 0x67, (byte) 0xF4,
+                (byte) 0x69, (byte) 0x63, (byte) 0x6F, (byte) 0x6E, (byte) 0x66, (byte) 0x69, (byte) 0x72, (byte) 0x6D, (byte) 0x65,
+                (byte) 0x64, (byte) 0xF4, (byte) 0x66, (byte) 0x61, (byte) 0x63, (byte) 0x74, (byte) 0x69, (byte) 0x76, (byte) 0x65,
+                (byte) 0xF4, (byte) 0x69, (byte) 0x70, (byte) 0x65, (byte) 0x72, (byte) 0x6D, (byte) 0x61, (byte) 0x6E, (byte) 0x65,
+                (byte) 0x6E, (byte) 0x74, (byte) 0xF4, (byte) 0xFF, (byte) 0xFF, (byte) 0x6B, (byte) 0x73, (byte) 0x70, (byte) 0x6C,
+                (byte) 0x69, (byte) 0x74, (byte) 0x53, (byte) 0x74, (byte) 0x61, (byte) 0x74, (byte) 0x75, (byte) 0x73, (byte) 0x00,
+                (byte) 0xFF};
+        try {
```

Comment:
Good practice in tests to mark you function with throwable and avoid `try {} catch {}` this will make your test cleaner. 
And if you test some exception is happeing so can mark `@Test(ExpectedException.class)